src/zjs_error.c

```diff
@@ -80,6 +80,63 @@ void zjs_error_cleanup()
     }
 }
 
+static char* construct_message(jerry_value_t this,
+                                      jerry_value_t func,
+                                      const char *message)
+{
+    jerry_size_t size = 64;
+    char name[size];
+    jerry_value_t keys_array = jerry_get_object_keys(this);
+    uint32_t arr_length = jerry_get_array_length(keys_array);
+    int i;
+    for (i = 0; i < arr_length; ++i) {
+        jerry_value_t val = jerry_get_property_by_index(keys_array, i);
+        zjs_copy_jstring(val, name, &size);
+        jerry_value_t func_val = zjs_get_property(this, name);
```

Comment:
I tried this and the function named ended up beings "X". Not sure why but it didn't seem to work.